foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/dynamic/DynamicClassLoader.java

```diff
@@ -96,17 +94,20 @@ public EclipseLinkClassWriter getClassWriter(String className) {
     }
 
     public void addEnum(String className, Object... literalLabels) {
+        var literalsMap = Arrays.stream(literalLabels)
```

Comment:
`var` is not preferred as we will backport this PR into 2.7 branch with JDK 8.0 support.